terraform/live/acm-particule.io/terragrunt.hcl

```diff
@@ -0,0 +1,47 @@
+include {
+  path = "${find_in_parent_folders()}"
+}
+
+terraform {
+  source = "github.com/terraform-aws-modules/terraform-aws-acm?ref=v2.12.0"
+}
+
+locals {
+  aws_region  = "us-east-1"
```

Comment:
eu-west-3 no ?

Reply:
For cloudfront the region is always us-east-1 for ACM